sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -288,9 +298,13 @@ export class SenderImpl implements Sender {
       sequenceNumbers
     );
 
-    return this._context.managementClient!.cancelScheduledMessages(sequenceNumbers);
+    return this._context.managementClient!.cancelScheduledMessages(
+      sequenceNumbers,
+      this._senderOptions
+    );
   }
 
+  // TODO - No need to retry close() ???
```

Comment:
At the moment `close()` doesnt have retry or cancellation token even in Event Hubs Track 2. Let's maintain that design for now

Reply:
Okay.

Reply:
So, can we remove the TODO?

Reply:
Done.